HousingSearchApi/V1/Boundary/Requests/GetAllAssetListRequest.cs

```diff
@@ -30,7 +30,8 @@ public class GetAllAssetListRequest : GetAssetListRequest
 
         [FromQuery(Name = "contractIsApproved")]
         public string ContractIsApproved { get; set; }
-
+        [FromQuery(Name = "contractIsActive")]
+        public string ContractIsActive { get; set; }
```

Comment:
Oh, okay, nevermind I think I remember having a chat with Miles about this.

This is that nonsensical portion of the application that's built to expect booleans as strings :man_facepalming: 

I suppose nothing that can be done here without a massive refactor. You can ignore previous comments.